### PR TITLE
Make country code not editable by passing prop countryCodeEditable

### PR DESCRIPTION
**Pull request by HasanShehryarJaffri**
Requirement:
User should not be able to remove country code by pressing backspace.

Solution:
Added a flag countryCodeEditable which is true by default so the existing users won't be able to feel any difference.
However, if this flag is passed as false, then the country code can't be removed by pressing backspaces.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
-    <meta name="apple-mobile-web-app-capable" content="yes">
-    <title>react-phone-input-test</title>
-  </head>
-  <body>
-    <div id="root"></div>
-    <script src="index.js"></script>
-  </body>
-</html>
```
Comment (bl00mber) [inline dist/index.html]: This file used for the demo and should not be deleted.

File: src/index.js [hunk 2]
```diff
@@ -154,7 +158,8 @@ class ReactPhoneInput extends React.Component {
       queryString: '',
       showDropdown: false,
       freezeSelection: false,
-      debouncedQueryStingSearcher: debounce(this.searchCountry, 100)
+      debouncedQueryStingSearcher: debounce(this.searchCountry, 100),
+      countryCodeEditable: props.countryCodeEditable
```
Comment (bl00mber) [inline src/index.js]: Please delete this and use prop directly from the props instead of appending it to the component state.

File: src/index.js [hunk 3]
```diff
@@ -425,6 +430,15 @@ class ReactPhoneInput extends React.Component {
     let newSelectedCountry = this.state.selectedCountry;
     let freezeSelection = this.state.freezeSelection;
 
+    const { countryCodeEditable } = this.state;
+    
+    if(!countryCodeEditable) {
```
Comment (bl00mber) [inline src/index.js]:
```js
if(!this.props.countryCodeEditable) {
```

File: dist/index.html (hunk 1 above)
Comment (HasanShehryarJaffri) [reply dist/index.html]: oh sorry. deleted that by mistake.

**Pull request merged**